specification/cognitiveservices/OpenAI.Inference/models/completions/chat_messages.tsp

```diff
@@ -182,6 +203,10 @@ model ChatResponseMessage {
   @doc("The chat role associated with the message.")
   role: ChatRole;
 
+  @added(ServiceApiVersions.v2024_08_01_Preview)
+  @doc("The refusal message generated by the model.")
+  refusal?: string;
```

Comment:
```suggestion
  refusal: string;
```
Refusal is required in OAI: https://github.com/openai/openai-openapi/blob/master/openapi.yaml#L9578